web/src/pages/Mountains.tsx

```diff
@@ -35,7 +35,8 @@ function MountainsPage(props: WithTranslation) {
     return (
         <main>
             <h1>{t('noun.mountains')}</h1>
-            <div className="grid">
+            <h1>Trails in:</h1>
+            <div className="grid trailsList">
```

Comment:
Was this change intentional? Looks like a leftover copy paste (both h1 and this div)